test/Constraints/members.swift

```diff
@@ -590,3 +590,36 @@ do {
   throw SR_2193_Error.Boom
 } catch let e as SR_2193_Error.Boom { // expected-error {{enum element 'Boom' is not a member type of 'SR_2193_Error'}}
 }
+
+// rdar://problem/25341015
+extension Sequence {
+  func r25341015_1() -> Int {
+    return max(1, 2) // expected-error {{type member 'max' shadows global function 'max' in module 'Swift', which cannot be called using implicit 'self.'}} {{12-12=Swift.}}
```

Comment:
IIRC from thinking about this before, the problem was that the conflicting member isn't *always* an instance member. It could be an instance member conflicting with a global function, or a static member conflicting with a global function, right?